src/test/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterControllerTest.java

```diff
@@ -46,21 +46,21 @@
     @Test
     public void should_return_message_id_when_letter_is_successfully_sent() throws Exception {
         given(tokenValidator.getServiceName("auth-header-value")).willReturn("service-name");
-        given(letterService.send(Mockito.any(Letter.class))).willReturn("12345");
+        given(letterService.send(any(Letter.class), anyString())).willReturn("12345");
 
         sendLetter(LETTER_JSON)
             .andExpect(status().isOk())
             .andExpect(content().string(containsString("12345")));
 
         verify(tokenValidator).getServiceName("auth-header-value");
-        verify(letterService).send(Mockito.any(Letter.class));
+        verify(letterService).send(any(Letter.class), anyString());
```

Comment:
In some place we could check if the controller passes the right service name to letter service.